src/services/template/UserResolvers.ts

```diff
@@ -131,4 +134,49 @@ export class UserResolver {
 
         return true;
     }
+
+    @Mutation(() => Boolean)
+    async resetPasswordToken(
+        @Arg("email") email: string
+    ) {
+        const buf = crypto.randomBytes(20)
+        const token = buf.toString('hex');
+
+        try {
+            const user = await User.findOne({ email: email });
+            user.resetPasswordToken = token;
+            user.resetPasswordExpires = 3600000; // 1 hour
+            await user.save()
+            const smtpTransport = nodemailer.createTransport({
+                service: 'Gmail', 
+                auth: {
+                  user: 'concordnoreply@gmail.com',
+                  pass: 'CONCORDFOOS123',
+                }
+            });
+
+            const mailOptions = {
+                to: user.email,
+                from: 'learntocodeinfo@gmail.com',
+                subject: 'Node.js Password Reset',
+                text: 'You are receiving this because you (or someone else) have requested the reset of the password for your account.\n\n' +
+                  'Please click on the following link, or paste this into your browser to complete the process:\n\n' +
+                  'http://' + 'localhost:3000' + '/resetpassword/' + token + '\n\n' +
```

Comment:
change `'http://' + 'localhost:3000'` to `process.env.NODE_ENV !== 'production' ? "http://localhost:3000" : "https://concord-app.herokuapp.com/"`